amundsen_application/static/js/components/common/Card/index.tsx

```diff
@@ -1,35 +1,41 @@
 import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import './styles.scss';
 
 export interface CardProps {
-  title: string;
+  title?: string;
```

Comment:
nit: Just wondering why all properties are optional, to be more generic/reuseable?

Reply:
It is a byproduct of integrating the loading state inside the card, as we won't have any of the properties available when loading them.